rapids/rapids.sh

```diff
@@ -12,7 +12,7 @@ readonly SPARK_VERSION_ENV=$(spark-submit --version 2>&1 | sed -n 's/.*version[[
 
 if [[ "${SPARK_VERSION_ENV}" == "3"* ]]; then
   readonly DEFAULT_CUDF_VERSION="0.17"
-  readonly DEFAULT_SPARK_RAPIDS_VERSION="0.3.0"
+  readonly DEFAULT_SPARK_RAPIDS_VERSION="0.1.0"
```

Comment:
Why this line is 0.1.0? I think it's the version of spark-rapids jar? Should be 0.3.0.

Reply:
Done.